test/e2e/data/infrastructure-vsphere-govmomi/main/ignition/kustomization.yaml

```diff
@@ -14,3 +14,10 @@ patches:
   - target:
       kind: VSphereMachineTemplate
     path: ../commons/remove-storage-policy.yaml
+  # Replace ssh user to match expected user by the e2e machine collector
```

Comment:
q: is it possible to change it cluster-template-ignition.yaml instead?

Reply:
I think we should keep the core user there. It's sort of standard with Ignition I think?

Reply:
Agree, `core` is the standard in ignition.